activerecord/lib/active_record/connection_adapters/abstract/schema_statements.rb

```diff
@@ -1223,6 +1225,10 @@ def extract_new_default_value(default_or_changes)
           default_or_changes
         end
       end
+
+      def can_remove_index_by_name?(options)
+        options.is_a?(Hash) && options.key?(:name) && options.except(:name, :algorithm).empty?
```

Comment:
options can be a Symbol http://apidock.com/rails/ActiveRecord/ConnectionAdapters/SchemaStatements/remove_index
and Symbol doesn't have `#key?` and `#except` methods